src/components/Alert/index.js

```diff
@@ -14,10 +14,15 @@ const iconMap = {
 /**
  * Component that is used to show alert messages on the UI
  */
-export const Alert = ({ className = '', type, message, ...props }) => {
+export const Alert = ({ className = '', type, message, variant, ...props }) => {
```

Comment:
We can give some default variant so that proper class name is added.
`variant='default'`